examples/ir_transmit.py

```diff
@@ -0,0 +1,25 @@
+"""IR transmit example using Circuit Playground Express"""
+import time
+import adafruit_irremote
+import pulseio
+import board
+import digitalio
+
+# Create a button object to trigger IR transmit
+button = digitalio.DigitalInOut(board.D4)
+button.direction = digitalio.Direction.INPUT
+# This is for CPX. For an external button
```

Comment:
Comment seems truncated?
"Using internal pull downs for CPX button compatibility. For other external buttons, either change or do the same."

Reply:
Oops. I got halfway though this idea and gave up on it, forgot to delete it. Fixing.